uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-calendar.scss

```diff
@@ -273,3 +238,59 @@ $chr-leave-calendar-legend-size:                  20px;
   margin-right: 20px;
   width: calc(100% - 50px);
 }
+
+// Legend
+
+.chr_leave-calendar__legend__title {
```

Comment:
Alphabetically legend should be before month, as it was before.

Reply:
Ordered ALL bems.